sdk/python/tests/integration/feature_repos/universal/feature_views.py

```diff
@@ -4,11 +4,44 @@
 from feast.data_source import DataSource
 
 
-def correctness_feature_view(data_source: DataSource) -> FeatureView:
+def driver_feature_view(
+    data_source: DataSource, name="test_correctness"
+) -> FeatureView:
     return FeatureView(
-        name="test_correctness",
+        name=name,
         entities=["driver"],
         features=[Feature("value", ValueType.FLOAT)],
         ttl=timedelta(days=5),
         input=data_source,
     )
+
+
+def create_driver_hourly_stats_feature_view(source):
+    driver_stats_feature_view = FeatureView(
+        name="driver_stats",
+        entities=["driver"],
+        features=[
+            Feature(name="conv_rate", dtype=ValueType.FLOAT),
```

Comment:
optional nit: we seem to reference these feature names + types across files. Could imagine a cleanup being having some field that's a map of feature_name -> Tuple[dtype, feature_ref(i.e. feature_view_name:feature_name), expected type] so we dont have to redefine the full set of features + their types.

Reply:
yeah there's probably some clean up we can do